main_test.go

```diff
@@ -68,12 +68,12 @@ func TestCheckForGitAndDockerHub(t *testing.T) {
 
 }
 func TestFilterApps(t *testing.T) {
-	test_input, err := ioutil.ReadFile("sample-golang.yaml")
+	testInput, err := ioutil.ReadFile("testdata/sample-golang.yaml")
```

Comment:
the `testdata` folder is ignored by the go tool, so we can put our tests data in there